lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -12,6 +12,36 @@ all_interfaces:
   get_context: ~
   get_value: '/^interface (.*)$/'
 
+global_ingress_replication_bgp:
+  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
+  os_version: 'N9k:9.2'
```

Comment:
This should be `N9k:9.1` right?

Reply:
I am not sure whether there is anything like 9.1
I know there is only 9.2.1 and 9.2.2
Please check again

Reply:
Good point 👍